modules/serverless-kafka-tls.adoc

```diff
@@ -0,0 +1,80 @@
+// Module is included in the following assemblies:
+//
+// serverless/serverless-kafka.adoc
+
+[id="serverless-configuring-tls-authentication-against-an-apache-kafka_{context}"]
+== Configuring TLS authentication
+
+.Prerequisites
+
+* A Kafka cluster CA certificate as a `.pem` file.
+* A Kafka cluster client certificate and key as `.pem` files.
+
+.Procedure
+
+. Create the certificate files as secrets in your chosen namespace:
++
+----
+$ kubectl create secret --namespace <namespace> generic <kafka-auth-secret> \
+  --from-file=ca.crt=caroot.pem \
+  --from-file=user.crt=certificate.pem \
+  --from-file=user.key=key.pem
+----
++
+[IMPORTANT]
+====
+Use the key names `ca.crt`, `user.crt`, and `user.key`. Do not change them.
+====
+
+
+. Start editing the `KnativeKafka` CR:
```

Comment:
"CR" isn't defined previously in the module - s/`KnativeKafka` CR/`KnativeKafka` custom resource/